bigchaindb/core.py

```diff
@@ -526,12 +536,15 @@ def write_block(self, block, durability='soft'):
         """
 
         block_serialized = rapidjson.dumps(block)
-        r.table('bigchain').insert(r.json(block_serialized), durability=durability).run(self.conn)
+        self.connection.run(
+                r.table('bigchain')
+                .insert(r.json(block_serialized), durability=durability))
 
     # TODO: Decide if we need this method
```

Comment:
I don't think we need this method. Its not being used anywhere and its behavior is now inconsistent with get_transaction.
This is just a note to remember to create an issue for this